EIPS/eip-5773.md

```diff
@@ -0,0 +1,362 @@
+---
+eip: 5773
+title: Multi-Resource Token
+description: An interface for Multi-Resource tokens.
```

Comment:
Try not to make this just a rewording of the title. It should expand a little beyond what's in the title.